### PR TITLE
Changed summary section text

### PR DESCRIPTION
**Pull request by benhowes**
### What is the context of this PR?
Text change for https://trello.com/c/kjjICyej/2192-change-all-summary-section-titles-to-check-your-answers-and-submit

### How to review 
Run `1_0005.json`, complete the survey as far as the final summary, the changes should make it look like: https://sketch.cloud/s/3Kk9r/all/pagewidth998px/checkanswersandsubmit/play (Other than the accordians which are in another card). Those changes are:
1. The right hand nav should now read "Check your answers and submit".
2. The "check your answers" panel below the title should now be blue.
3. There are a couple of minor text changes to match the designs.

### CODE REVIEW

**Review by MebinAbraham — DISMISSED** [review]

**Review by dcdarrell9 — DISMISSED** [review]

**Comment by benhowes** [comment]
Pending clarification that this was all that was required!

**Comment by benhowes** [comment]
@MebinAbraham @dcdarrell9 I've made some additional changes, can you re-review?

**Review by insacuri — APPROVED** [review]
LGTM

File: app/templates/summary.html [hunk 1]
```diff
@@ -16,12 +16,13 @@
...
-  <h1 class="saturn">{{ _("Your answers") }}</h1>
+  <h1 class="saturn">{{ _("Check your answers and submit") }}</h1>
   <div class="print__message panel panel--simple panel--error u-mb-l">
     <h2 class="saturn">{{ _("Please remember to submit these answers") }}.</h2>
   </div>
-  <div class="print__hidden panel panel--simple panel--warn u-mb-l">
-    <h2 class="neptune">{{ _("Please check your answers carefully before submitting") }}.</h2>
+  <div class="print__hidden panel panel--simple panel--info u-mb-l">
+    <strong>{{ _("Please submit this survey to complete it") }}</strong>
+    <p>{{ _("You can check your answers below") }}</p>
```
Comment (jonnyshaw89) [inline app/templates/summary.html:25]: This change isn't explicitly detailed in the card, but i'm assuming it was desired?

**Review by jonnyshaw89 — APPROVED** [review]

File: app/templates/summary.html (hunk 1 above)
Comment (benhowes) [reply app/templates/summary.html:25]: Yeah, that was the change in point 3 - It should look the same as the prototype linked

Comment (jonnyshaw89) [reply app/templates/summary.html:25]: point 3?

Comment (benhowes) [reply app/templates/summary.html:25]: From the PR description:
> 3. There are a couple of minor text changes to match the designs.

Comment (jonnyshaw89) [reply app/templates/summary.html:25]: ah, i was using the Trello card for reference

**Comment by benhowes** [comment]
Pending a fix over in the schema-validator.

File: app/templates/summary.html [hunk 2]
```diff
@@ -16,12 +16,13 @@
 
 {% if content.summary.summary_type == 'Summary' %}
 <div>
-  <h1 class="saturn">{{ _("Your answers") }}</h1>
+  <h1 class="saturn">{{ _("Check your answers and submit") }}</h1>
   <div class="print__message panel panel--simple panel--error u-mb-l">
     <h2 class="saturn">{{ _("Please remember to submit these answers") }}.</h2>
   </div>
```
Comment (MebinAbraham) [inline app/templates/summary.html:22]: Feel like this should have always been `your` not `these`.

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**